index.js

```diff
@@ -2746,6 +2748,59 @@
         return filter(is(type), Just(x));
       });
 
+  //# keys :: StrMap a -> Array String
+  //.
+  //. Returns an array of the keys of the supplied StrMap.
+  //.
+  //. ```javascript
+  //. > S.keys({a: 1, b: 2, c: 3})
+  //. ['a', 'b', 'c']
+  //. ```
+  S.keys =
+  def('keys',
+      {},
+      [$.StrMap(a), $.Array($.String)],
+      function(strMap) { return Object.keys(strMap); });
```

Comment:
Is this anonymous function necessary? It seems we could use `Object.keys` directly.